recipe/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
     - cmake
     - ninja
     - fypp
+    - {{ mpi }}  # [mpi == "openmpi" and build_platform != target_platform]
```

Comment:
Stupid questions, but why the restriction on openmpi here? Wouldn't that -- in theory -- also be needed for all MPI frameworks? (Or for none of them, as it is already listed in the `host` section? I somewhat fail to understand, which of the two would be the natural block to contain `{{ mpi }}`...)

Reply:
OpenMPI is special, we need the MPI wrappers for the build platform to run the cross-compilers, otherwise CMake can't detect MPI correctly.

Reply:
The MPI wrappers from MPICH run just fine with cross-compilers.